WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -94,6 +155,82 @@ function WeakAuras.regionPrototype.create(region)
   region.RunCode = RunCode;
 end
 
+-- SetDurationInfo
+
+function WeakAuras.regionPrototype.modify(parent, region, data)
+  local defaultsForRegion = WeakAuras.regionTypes[data.regionType] and WeakAuras.regionTypes[data.regionType].default;
+  local hasAdjustedMin = defaultsForRegion and defaultsForRegion.useAdjustededMin ~= nil and data.useAdjustededMin;
+  local hasAdjustedMax = defaultsForRegion and defaultsForRegion.useAdjustededMax ~= nil and data.useAdjustededMax;
+
+  region.adjustedMin = hasAdjustedMin and data.adjustedMin and data.adjustedMin > 0 and data.adjustedMin;
+  region.adjustedMax = hasAdjustedMax and data.adjustedMax and data.adjustedMax > 0 and data.adjustedMax;
+end
+
+function WeakAuras.SetProgressValue(region, value, total)
```

Comment:
We do that all over the WeakAuras code, e.g. see Prototypes.lua or Types.lua. In this case, the function could probably be local though.